testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -919,8 +919,8 @@ void testCastIntervalToInterval(CastType castType, SqlOperatorFixture f) {
   void testCastWithRoundingToScalar(CastType castType, SqlOperatorFixture f) {
     f.setFor(SqlStdOperatorTable.CAST, VmName.EXPAND);
 
-    f.checkFails("cast(1.25 as int)", "INTEGER", true);
-    f.checkFails("cast(1.25E0 as int)", "INTEGER", true);
+    f.checkScalar("cast(1.25 as int)", 1, "INTEGER NOT NULL");
```

Comment:
The new version of this test looks correct to me. I wonder why the original test was expected to fail.